index.html

```diff
@@ -0,0 +1,857 @@
+            <li><a href="#">Amas</a></li>
+            <li><a href="#">Spaces</a></li>
+            <li><a href="#">Terms & Condition</a></li>
+          </ul>
+        </div>
+        <div class="nav-right">
+          <ul class="ul-black-nav">
+            <li><a href="#">About</a></li>
+            <li><a href="#">Advertise</a></li>
+            <li><a href="#">Jobs</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </div>
+      </div>
+      <div class="fix-nav">
+        <div class="fix-nav-left">
+          <img class="logo" src="images/the-next-web-tnw-logo.png" alt="logo" />
+          <!-- ++++++++++++++++++++++++++++++ Displays on Ipad and Desktop+++++++++++++++++++++++++  -->
+          <div class="white-nav-links">
+            <ul class="ul-white-nav">
+              <li><a href="#">latest</a></li>
+              <li><a href="#">hard fork</a></li>
+              <li><a href="#">plugged</a></li>
+              <li><a href="#">readme</a></li>
+              <li><a href="#">growth quateres</a></li>
+              <li><a href="#">neural</a></li>
+            </ul>
+          </div>
+        </div>
+
+        <div class="fix-nav-right">
+          <div class="search-img">
+            <i class="fas fa-search"></i>
+          </div>
+          <div class="hamburger">
+            <i class="fas fa-bars"></i>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <div class="sweatpants">
+      We heard you like events and sweatpant so we made event ...
+      <span class="disappear">you can join in your sweatpant ➔</span>
+    </div>
+
+    <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        Main Section
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+    <main class="main-section">
+      <div class="main-wrapper">
+        <div class="header-img hero-img-1 span-2-rows desk-header-img">
+          <div class="overlay">
+            <p class="img-text text-bg">
+              Netflix will cancel account if you havent used it in a while
+            </p>
+          </div>
+        </div>
+        <div class="header-img hero-img-2 desk-header-img2">
+          <div class="overlay">
+            <p class="img-text text-bg-sm">
+              Shopify, Base camp & more share whats next for digital products on
+              June 11
+            </p>
+          </div>
+        </div>
+        <div class="header-img hero-img-3 desk-header-img2">
+          <p class="img-text text-bg-sm-3">What is Chademo? Let us explain</p>
+        </div>
+      </div>
+
+      <!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        Latest News Section
+      ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ -->
+
+      <section class="latest-news">
+        <h2 class="heading2 orange">Latest News ></h2>
```

Comment:
- [ ] Kindly check the layout for the latest news section for medium devices.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83333975-08bb9b80-a269-11ea-9688-05b361a006d6.png)  | ![image](https://user-images.githubusercontent.com/57414789/83333958-e7f34600-a268-11ea-9b7f-10867ae33d84.png)